ui/src/main.css

```diff
@@ -116,3 +116,100 @@ input[type='color']:focus,
 .nowrap-style {
   white-space: nowrap;
 }
+
+.rc-input-number {
```

Comment:
I tried importing `rc-input-number`'s default styles but it required a lot of overrides to make it look the way I wanted, so instead I decided to duplicate the styles and tweak them to my liking. It's not a lot of styles overall, so I think this is more maintainable that the former solution.